RadarSDK/RadarSettings.m

```diff
@@ -227,6 +228,19 @@ + (void)setFeatureSettings:(RadarFeatureSettings *)featureSettings {
     }
 }
 
++ (RadarSDKConfiguration *)sdkConfiguration {
+    RadarSDKConfiguration * sdkConfiguration = [[RadarSDKConfiguration alloc] init];
+    sdkConfiguration.logLevel = [RadarSettings logLevel];
+    return sdkConfiguration;
+}
+
++ (void)setSDKConfiguration:(RadarSDKConfiguration *)sdkConfiguration {
+    [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug 
+        message:[NSString stringWithFormat:@"Setting SDK Configuration | sdkConfiguration = %@", sdkConfiguration]];
+    if (sdkConfiguration) {
+        [RadarSettings setLogLevel:sdkConfiguration.logLevel];
+    }
```

Comment:
Hmmmm, I think instead of writing to log level like `[RadarSettings setLogLevel:...]`, maybe we should just write to a `sdkConfiguration` object stored in `NSUserDefaults` (like we do for feature settings), and then in `[RadarSettings logLevel]`, we implement as the second conditional check (`RadarSettings userDebug` should remain the first) where we use the `sdkConfiguration.logLevel` if it exists, otherwise falling back to the local like we do now.